src/modules/builtin/mv.rs

```diff
@@ -0,0 +1,40 @@
+use heraclitus_compiler::prelude::*;
+use crate::modules::expression::expr::Expr;
+use crate::modules::condition::failed::Failed;
+use crate::translate::module::TranslateModule;
+use crate::utils::{ParserMetadata, TranslateMetadata};
+
+#[derive(Debug, Clone)]
+pub struct Mv {
+    source: Expr,
+    destination: Expr,
+    failed: Failed,
+}
+
+impl SyntaxModule<ParserMetadata> for Mv {
+    syntax_name!("MoveFiles");
+
+    fn new() -> Self {
+        Mv {
+            source: Expr::new(),
+            destination: Expr::new(),
+            failed: Failed::new(),
+        }
+    }
+
+    fn parse(&mut self, meta: &mut ParserMetadata) -> SyntaxResult {
+        token(meta, "mv")?;
+        syntax(meta, &mut self.source)?;
+        syntax(meta, &mut self.destination)?;
+        syntax(meta, &mut self.failed)?;
+        Ok(())
+    }
+}
+
+impl TranslateModule for Mv {
+    fn translate(&self, meta: &mut TranslateMetadata) -> String {
+        let source = self.source.translate(meta);
+        let destination = self.destination.translate(meta);
+        format!("mv {} {} || exit", source, destination)
```

Comment:
What is that `|| exit`? This will make the code execution to stop when error happens?

Reply:
I think that is part of a copy and paste of #311 that use that to align with shellcheck.
But I don't find a rule for mv so I think that can be removed.

Reply:
added the functionality to parse Command Modifiers, Please review the changes and let me know if there are any further improvements or adjustments needed